install_spyder.py

```diff
@@ -0,0 +1,150 @@
+
+    REPOS[p.name] = {'repo': p, 'dist': dist, 'editable': editable}
+
+# ---- Setup logger
+fmt = Formatter('%(asctime)s [%(levelname)s] [%(name)s] -> %(message)s')
+h = StreamHandler()
+h.setFormatter(fmt)
+logger = getLogger('InstallSubRepos')
+logger.addHandler(h)
+logger.setLevel('INFO')
+
+
+def get_python_lsp_version():
+    """Get current version to pass it to setuptools-scm."""
+    req_file = DEVPATH / 'requirements' / 'conda.txt'
+    with open(req_file, 'r', encoding='utf-8') as f:
+        for line in f:
+            if 'python-lsp-server' not in line:
+                continue
+
+            version = next(iter(Requirement(line).specifier)).version
+
+            break
+
+    return version
+
+
+def install_repo(name, editable=False):
+    """
+    Install a single repo from source located in spyder/external-deps, ignoring
+    dependencies, in standard or editable mode.
+
+    Parameters
+    ----------
+    name : str
+        Must be 'spyder' or the distribution name of a repo in
+        spyder/external-deps.
+    editable : bool (False)
+        Install repo in editable mode (True) or standard mode (False).
+        This uses the `-e` flag.
+
+    """
+    try:
+        repo_path = REPOS[name]['repo']
+    except KeyError:
+        logger.warning(
+            'Distribution %r not valid. Must be one of %s', name, set(REPOS.keys()))
+        return
+
+    install_cmd = BASE_COMMAND.copy()
+
+    # PyLSP requires pretend version
+    env = None
+    if name == 'python-lsp-server':
+        env = {**os.environ}
+        env.update({'SETUPTOOLS_SCM_PRETEND_VERSION': get_python_lsp_version()})
+
+    if editable:
+        # Add edit flag to install command
+        install_cmd.append('-e')
+        mode = 'editable'
+    else:
+        mode = 'standard'
+
+    logger.info('Installing %r from source in %s mode.', name, mode)
+    install_cmd.append(repo_path.as_posix())
+    check_output(install_cmd, env=env)
+
+
+def main(install=tuple(REPOS.keys()), no_install=tuple(), **kwargs):
+    """
+    Install all subrepos from source.
+
+    Parameters
+    ----------
+    install : iterable (spyder and all repos in spyder/external-deps)
+        Distribution names of repos to be installed from spyder/external-deps.
+    no_install : iterable ()
+        Distribution names to exclude from install.
+    **kwargs :
+        Keyword arguments passed to `install_repo`.
+
+    """
+    _install = set(install) - set(no_install)
+    for repo in _install:
+        install_repo(repo, **kwargs)
+
+
+if __name__ == '__main__':
+    # ---- Parse command line
+
+    parser = argparse.ArgumentParser(
+        usage="python install_spyder.py [options]")
+    parser.add_argument(
+        '--no-install', dest='no_install', nargs='+', default=[],
+        help="Space-separated list of distribution names to exclude from "
+             "install. Default is empty list."
+    )
+    parser.add_argument(
+        '--install', dest='subrepos', nargs='+',
```

Comment:
```suggestion
        '--install', dest='install', nargs='+',
```

Looks like this wasn't updated, causing the CI failures—FYI, if you omit the `dest` argument on all of these, it will automatically use the argument name, so you don't have to update it two places (which leads to bugs like this).